lib/docker-delta.coffee

```diff
@@ -136,8 +150,8 @@ exports.applyDelta = (srcImage, dstImage) ->
 			# rsync doesn't fsync by itself
 			utils.waitPidAsync(spawn('sync'))
 		.catch (e) ->
-			if code in DELTA_OUT_OF_SYNC_CODES
-				throw new OutOfSyncError('Incompatible image')
+			if e?.code in DELTA_OUT_OF_SYNC_CODES
+				throw new Error('Incompatible image')
```

Comment:
Throwing the `OutOfSyncError` meant it was really easy to catch and handle specially via
```
.catch delta.OutOfSyncError, (e) ->
	...
```
It'd be really nice to keep that easy rather than having to do `if e.message is 'Incompatible image'`